
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../installation/">
      
      
        <link rel="next" href="../KineticHub/">
      
      <link rel="icon" href="../assets/RAP_logo.ico">
      <meta name="generator" content="mkdocs-1.5.1, mkdocs-material-9.1.21">
    
    
      
        <title>RAP Builder - RAP</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.eebd395e.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.ecc896b0.min.css">
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Barlow:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Barlow";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    
      <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script>
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#rap-builder" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow md-header--lifted" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="RAP" class="md-header__button md-logo" aria-label="RAP" data-md-component="logo">
      
  <img src="../assets/RAP_logo.ico" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            RAP
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              RAP Builder
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
          
            
            
            
            <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
            
              <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_2" hidden>
                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
              </label>
            
          
            
            
            
            <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_2">
            
              <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12c0-2.42-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
              </label>
            
          
        </form>
      
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
    
      
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        Home
      </a>
    </li>
  

      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href="../get_started/" class="md-tabs__link">
        User Guide
      </a>
    </li>
  

      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href="../installation/" class="md-tabs__link">
        Installation
      </a>
    </li>
  

      
        
  
  
    
  


  
  
  
    <li class="md-tabs__item">
      <a href="./" class="md-tabs__link md-tabs__link--active">
        How RAP works
      </a>
    </li>
  

      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href="https://2023.igem.wiki/fudan/software" class="md-tabs__link">
        Wiki
      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
    
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="RAP" class="md-nav__button md-logo" aria-label="RAP" data-md-component="logo">
      
  <img src="../assets/RAP_logo.ico" alt="logo">

    </a>
    RAP
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_1" >
      
      
      
        <label class="md-nav__link" for="__nav_1" id="__nav_1_label" tabindex="0">
          Home
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_1_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_1">
          <span class="md-nav__icon md-icon"></span>
          Home
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        RAP Documentation
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
      
      
      
        <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
          User Guide
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          User Guide
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../get_started/" class="md-nav__link">
        Quickstart of RAP
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../advanced/" class="md-nav__link">
        Advanced
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
      
      
      
        <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
          Installation
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          Installation
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../installation/" class="md-nav__link">
        Installation
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
    
  
  
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" checked>
      
      
      
        <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
          How RAP works
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="true">
        <label class="md-nav__title" for="__nav_4">
          <span class="md-nav__icon md-icon"></span>
          How RAP works
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          RAP Builder
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        RAP Builder
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#introduction" class="md-nav__link">
    Introduction
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prap-system" class="md-nav__link">
    pRAP system
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#establishing-rbs-strength" class="md-nav__link">
    Establishing RBS strength
  </a>
  
    <nav class="md-nav" aria-label="Establishing RBS strength">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#assumption-1" class="md-nav__link">
    Assumption 1
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assumption-2" class="md-nav__link">
    Assumption 2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assumption-3" class="md-nav__link">
    Assumption 3
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#finding-rbs-with-optimal-intensity" class="md-nav__link">
    Finding RBS with optimal intensity
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#references" class="md-nav__link">
    References
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../KineticHub/" class="md-nav__link">
        KineticHub
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
      
      
      
        <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
          Wiki
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_5">
          <span class="md-nav__icon md-icon"></span>
          Wiki
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="https://2023.igem.wiki/fudan/software" class="md-nav__link">
        None
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#introduction" class="md-nav__link">
    Introduction
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prap-system" class="md-nav__link">
    pRAP system
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#establishing-rbs-strength" class="md-nav__link">
    Establishing RBS strength
  </a>
  
    <nav class="md-nav" aria-label="Establishing RBS strength">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#assumption-1" class="md-nav__link">
    Assumption 1
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assumption-2" class="md-nav__link">
    Assumption 2
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assumption-3" class="md-nav__link">
    Assumption 3
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#finding-rbs-with-optimal-intensity" class="md-nav__link">
    Finding RBS with optimal intensity
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#references" class="md-nav__link">
    References
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


<h1 id="rap-builder">RAP Builder</h1>
<h2 id="introduction">Introduction</h2>
<p>In synthetic biology, multipart reactions of exogenous import often present a conflict between reaction efficiency and problems such as excessive metabolic stress and metabolic stress in cells. If too low an amount of an enzyme is expressed in a multi-step reaction, the rate-limiting step will result in an inefficient reaction, but if too high an enzyme is expressed in a multi-part reaction, it will result in increased metabolic pressure on the cell due to excessive consumption of the carbon source as well as metabolic stress due to the buildup of the substrate. Therefore a suitably tunable multigene co-expression system is necessary.</p>
<h2 id="prap-system">pRAP system</h2>
<p>There are two ways to implement multigene co-expression system,which are the multi-promoter co-expression strategy and the polycistronic co-expression strategy. Compared to the multi-promoter co-expression strategy, the polycistronic co-expression strategy is more suitable for synthetic biology because it requires minimal cargo DNA to deliver multiple genes and all genes complete their expression under one promoter<sup id="fnref:1"><a class="footnote-ref" href="#fn:1">1</a></sup>. Polycistronic expression also offers reproducible ratios of the proteins expressed, minimizing the impact on protein expression of the fluctuating transcription associated with multiple promoters because all genes are transcriptionally linked<sup id="fnref:2"><a class="footnote-ref" href="#fn:2">2</a></sup>. However, a major problem with polycistronic expression, which has two or more target genes under one promoter, is that the downstream genes express significantly lower than the first gene next to the promoter<sup id="fnref:3"><a class="footnote-ref" href="#fn:3">3</a></sup><sup>,</sup><sup id="fnref:4"><a class="footnote-ref" href="#fn:4">4</a></sup>.</p>
<p>To solve this problem, our team proposed the Ribozyme-Assisted Polycistronic co-expression system (pRAP system) last year and won the 2022 Best New Basic Part. We develop pRAP system by sandwiching ribozyme sequences between CDSs instead of sequentially building CDSs. In pRAP, the hammerhead ribozyme's RNA sequences execute self-cleaving, leading in the co-transcriptional conversion of the polycistronic mRNA transcript into separate mono-cistrons <em>in vivo</em>.</p>
<h2 id="establishing-rbs-strength">Establishing RBS strength</h2>
<p>Even though we discovered and demonstrated the pRAP system's functionality last year, we were only able to measure whether or not the RBS intensity needed to be increased or decreased. This year, we proposed a method for quantitatively evaluating optimal RBS intensity. Our method focuses on the balanced state of cellular metabolism and is based on the following assumptions:</p>
<h3 id="assumption-1">Assumption 1</h3>
<p>The fluxes are balanced between each step of the linear reaction and can be expressed as follows:</p>
<div class="arithmatex">\[
flux_1 = flux_2 = ... = flux_n
\]</div>
<p>where <span class="arithmatex">\(n\)</span> is the index of the reaction and <span class="arithmatex">\(flux\)</span> means flux of the reaction. The significance of this assumption is that it prevents metabolic stress caused by flux imbalance and permits the cell to be in a state of flux balance.</p>
<h3 id="assumption-2">Assumption 2</h3>
<p>The substrate should not be overstacked and should be kept more stationary, which we denote here as follows:</p>
<div class="arithmatex">\[
\frac{[S_1]}{k_{m,1}} = \frac{[S_2]}{k_{m,2}} = ... = \frac{[S_n]}{k_{m,n}}
\]</div>
<p>where <span class="arithmatex">\([S_n]\)</span> represents the concentration of the substrate of <span class="arithmatex">\(reaction_n\)</span> and <span class="arithmatex">\(k_{m,n}\)</span> represents the <span class="arithmatex">\(k_m\)</span> value of reaction <span class="arithmatex">\(n\)</span>.</p>
<p>From the Michaelis-Menten equation we could find:</p>
<div class="arithmatex">\[
v_n = \frac{k_{cat,n}c_n[S_n]}{k_{m,n}+[S_n]}=\frac{k_{cat,n}c_n}{\frac{k_{m,n}}{[S_n]}+1}
\]</div>
<p>Where <span class="arithmatex">\(c_n\)</span> is the concentration of the enzyme of <span class="arithmatex">\(reaction_n\)</span> and <span class="arithmatex">\(k_{cat,n},v_n\)</span> is the <span class="arithmatex">\(k_{cat}\)</span> and velocity value of <span class="arithmatex">\(reaction_n\)</span>.</p>
<p>and</p>
<div class="arithmatex">\[
flux_n = v_n \times A_n
\]</div>
<p>where <span class="arithmatex">\(A_n\)</span> is the ratio of product to substrate stoichiometry.</p>
<p>Combining these equations and Assumption 1 we can conclude:</p>
<div class="arithmatex">\[
c_n\propto\frac{1}{A_nk_{cat,n}}
\]</div>
<p>From KineticHub we can easily search for <span class="arithmatex">\(k_{cat}\)</span>. By obtaining the <span class="arithmatex">\(A_n\)</span> as well as the <span class="arithmatex">\(k_{cat}\)</span> for each step in the linear reaction, we know what the ratio of concentration between the enzymes in each step is. </p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>It is important to note that the purpose of regulating RBS strength here is to regulate the relative proportional coordination between exogenously imported reactions, and the regulation of the absolute strength of the linear reaction coupled to intracellular metabolism is controlled by the induction intensity.</p>
</div>
<h3 id="assumption-3">Assumption 3</h3>
<p>The pRAP system introduces a stem-loop to avoid degradation of RNA after self-cleaving so the rate of RNA degradation is significantly reduced.<sup id="fnref:5"><a class="footnote-ref" href="#fn:5">5</a></sup> And the protein concentration in the cell can be expressed as follows<sup id="fnref:6"><a class="footnote-ref" href="#fn:6">6</a></sup>:</p>
<div class="arithmatex">\[
c_n = \frac{\alpha_{p,n}}{\delta_n} \cdot \frac{\alpha_{m,n}[G_n]}{\lambda}
\]</div>
<p>where <span class="arithmatex">\(\alpha_{p,n}\)</span> is the translation initiation rate of <span class="arithmatex">\(reaction_n\)</span>, <span class="arithmatex">\(\delta_n\)</span> is the mRNA degradation rate of <span class="arithmatex">\(reaction_n\)</span>, <span class="arithmatex">\(\alpha_{m,i}\)</span> is the transcription initiation rate of <span class="arithmatex">\(reaction_n\)</span>, <span class="arithmatex">\([G_n]\)</span> is the gene concentration of <span class="arithmatex">\(reaction_n\)</span>, <span class="arithmatex">\(\lambda\)</span> is the growth rate of the cell. Since the enzymes under pRAP system are all expressed under the same promoter, the gene concentration and transcription initiation rate are the same between reactions:</p>
<div class="arithmatex">\[
[G_1] = [G_{2}] = ... = [G_n]
\]</div>
<div class="arithmatex">\[
\alpha_{m,1} = \alpha_{m,2} = ... = \alpha_{m,n}
\]</div>
<p>And since the mRNA degradation rate in the presence of the stem-loop were all low, we concluded that there was little difference in the rates of RNA degradation between the reactions:</p>
<div class="arithmatex">\[
\delta_1 \approx \delta_2 \approx...\approx \delta_n
\]</div>
<p>Based on this assumption, we can notice:</p>
<div class="arithmatex">\[
\alpha_{p,n}\propto\frac{1}{A_nk_{cat,n}}
\]</div>
<p>By calculating the transcription initiation rate we can get the RBS intensity of the optimal.</p>
<h2 id="finding-rbs-with-optimal-intensity">Finding RBS with optimal intensity</h2>
<p>After getting the optimal RBS intensity, we need to get a synthetic RBS of that intensity (TIR rate). We used a thermodynamic approach combined with a Monte Carlo algorithm to obtain the optimal RBS<sup id="fnref:7"><a class="footnote-ref" href="#fn:7">7</a></sup><sup>,</sup><sup id="fnref:8"><a class="footnote-ref" href="#fn:8">8</a></sup>.</p>
<p>Transcription initiation rate can be expressed in terms of the Gibbs free energy change of transcription<sup id="fnref2:7"><a class="footnote-ref" href="#fn:7">7</a></sup>:</p>
<div class="arithmatex">\[
\alpha_{p,n} = Ke^{-\beta\Delta G_{total,n}}
\]</div>
<p>where <span class="arithmatex">\(K\)</span> and <span class="arithmatex">\(\beta\)</span> is a constant can be established experimentally, <span class="arithmatex">\(\Delta G_{total,n}\)</span> represents the Gibbs free energy change of <span class="arithmatex">\(reaction_n\)</span>. <span class="arithmatex">\(\Delta G_{total,n}\)</span> could  be expressed by the following equation:</p>
<div class="arithmatex">\[
\Delta G_{total,n} = \Delta G_{final,n} - \Delta G_{initial,n}\\= \Delta G_{mRNA:rRNA,n} + \Delta G_{start,n} + \Delta G_{spacing,n} - \Delta G_{standby,n} - \Delta G_{mRNA,n}
\]</div>
<p>The <span class="arithmatex">\(\Delta G_{mRNA:rRNA,n}\)</span>, <span class="arithmatex">\(\Delta G_{spacing,n}\)</span>, <span class="arithmatex">\(\Delta G_{standby,n}\)</span>, <span class="arithmatex">\(\Delta G_{mRNA,n}\)</span> in this equation is calculated by ViennaRNA<sup id="fnref:9"><a class="footnote-ref" href="#fn:9">9</a></sup>. For a defined start codon, <span class="arithmatex">\(\Delta G_{start,n}\)</span> is identified.</p>
<p>We use an Monte Carlo algorithm to find optimal RBS. The schematic of the algorithm is shown below:</p>
<p><img alt="" src="../assets/monte_carlo.svg" /></p>
<h2 id="references">References</h2>
<div class="footnote">
<hr />
<ol>
<li id="fn:1">
<p>Renaud-Gabardos, E., Hantelys, F., Morfoisse, F., Chaufour, X., Garmy-Susini, B., &amp; Prats, A.-C. (2015). Internal ribosome entry site-based vectors for combined gene therapy. <em>World Journal of Experimental Medicine</em>, <em>5</em>(1), 11–20. https://doi.org/10.5493/wjem.v5.i1.11&#160;<a class="footnote-backref" href="#fnref:1" title="Jump back to footnote 1 in the text">&#8617;</a></p>
</li>
<li id="fn:2">
<p>Sokolenko, S., George, S., Wagner, A., Tuladhar, A., Andrich, J. M. S., &amp; Aucoin, M. G. (2012). Co-expression vs. co-infection using baculovirus expression vectors in insect cell culture: Benefits and drawbacks. <em>Biotechnology Advances</em>, <em>30</em>(3), 766–781. https://doi.org/10.1016/j.biotechadv.2012.01.009&#160;<a class="footnote-backref" href="#fnref:2" title="Jump back to footnote 2 in the text">&#8617;</a></p>
</li>
<li id="fn:3">
<p>Qi, L., Haurwitz, R. E., Shao, W., Doudna, J. A., &amp; Arkin, A. P. (2012). RNA processing enables predictable programming of gene expression. <em>Nature Biotechnology</em>, <em>30</em>(10), Article 10. https://doi.org/10.1038/nbt.2355&#160;<a class="footnote-backref" href="#fnref:3" title="Jump back to footnote 3 in the text">&#8617;</a></p>
</li>
<li id="fn:4">
<p>Kim, K.-J., Kim, H.-E., Lee, K.-H., Han, W., Yi, M.-J., Jeong, J., &amp; Oh, B.-H. (2004). Two-promoter vector is highly efficient for overproduction of protein complexes. <em>Protein Science: A Publication of the Protein Society</em>, <em>13</em>(6), 1698–1703. https://doi.org/10.1110/ps.04644504&#160;<a class="footnote-backref" href="#fnref:4" title="Jump back to footnote 4 in the text">&#8617;</a></p>
</li>
<li id="fn:5">
<p>Liu, Y., Wu, Z., Wu, D., Gao, N., &amp; Lin, J. (2023). Reconstitution of Multi-Protein Complexes through Ribozyme-Assisted Polycistronic Co-Expression. <em>ACS Synthetic Biology</em>, <em>12</em>(1), 136–143. https://doi.org/10.1021/acssynbio.2c00416&#160;<a class="footnote-backref" href="#fnref:5" title="Jump back to footnote 5 in the text">&#8617;</a></p>
</li>
<li id="fn:6">
<p>Balakrishnan, R., Mori, M., Segota, I., Zhang, Z., Aebersold, R., Ludwig, C., &amp; Hwa, T. (2022). Principles of gene regulation quantitatively connect DNA to RNA and proteins in bacteria. <em>Science (New York, N.Y.)</em>, <em>378</em>(6624), eabk2066.&#160;<a class="footnote-backref" href="#fnref:6" title="Jump back to footnote 6 in the text">&#8617;</a></p>
</li>
<li id="fn:7">
<p>Salis, H. M., Mirsky, E. A., &amp; Voigt, C. A. (2009). Automated design of synthetic ribosome binding sites to control protein expression. <em>Nature Biotechnology</em>, <em>27</em>(10), Article 10. https://doi.org/10.1038/nbt.1568&#160;<a class="footnote-backref" href="#fnref:7" title="Jump back to footnote 7 in the text">&#8617;</a><a class="footnote-backref" href="#fnref2:7" title="Jump back to footnote 7 in the text">&#8617;</a></p>
</li>
<li id="fn:8">
<p>Salis, H. (2023). <em>Hsalis/Ribosome-Binding-Site-Calculator-v1.0</em> [Python]. https://github.com/hsalis/Ribosome-Binding-Site-Calculator-v1.0 (Original work published 2009)&#160;<a class="footnote-backref" href="#fnref:8" title="Jump back to footnote 8 in the text">&#8617;</a></p>
</li>
<li id="fn:9">
<p>Lorenz, R., Bernhart, S. H., Höner zu Siederdissen, C., Tafer, H., Flamm, C., Stadler, P. F., &amp; Hofacker, I. L. (2011). ViennaRNA Package 2.0. <em>Algorithms for Molecular Biology</em>, <em>6</em>(1), 26. https://doi.org/10.1186/1748-7188-6-26&#160;<a class="footnote-backref" href="#fnref:9" title="Jump back to footnote 9 in the text">&#8617;</a></p>
</li>
</ol>
</div>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2023 Zhiyue Chen
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
    
    
    <a href="mailto:20301050198@fudan.edu.cn" target="_blank" rel="noopener" title="" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4l217.6 163.2c11.4 8.5 27 8.5 38.4 0l217.6-163.2c12.1-9.1 19.2-23.3 19.2-38.4 0-26.5-21.5-48-48-48H48zM0 176v208c0 35.3 28.7 64 64 64h384c35.3 0 64-28.7 64-64V176L294.4 339.2a63.9 63.9 0 0 1-76.8 0L0 176z"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://igem-community.slack.com/team/U046NMGM7T4" target="_blank" rel="noopener" title="igem-community.slack.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://gitlab.igem.org/2023/software-tools/fudan" target="_blank" rel="noopener" title="gitlab.igem.org" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="m503.5 204.6-.7-1.8-69.7-181.78c-1.4-3.57-3.9-6.59-7.2-8.64-2.4-1.55-5.1-2.515-8-2.81-2.9-.295-5.7.083-8.4 1.11-2.7 1.02-5.1 2.66-7.1 4.78-1.9 2.12-3.3 4.67-4.1 7.44l-47 144H160.8l-47.1-144c-.8-2.77-2.2-5.31-4.1-7.43-2-2.12-4.4-3.75-7.1-4.77a18.1 18.1 0 0 0-8.38-1.113 18.4 18.4 0 0 0-8.04 2.793 18.09 18.09 0 0 0-7.16 8.64L9.267 202.8l-.724 1.8a129.57 129.57 0 0 0-3.52 82c7.747 26.9 24.047 50.7 46.447 67.6l.27.2.59.4 105.97 79.5 52.6 39.7 32 24.2c3.7 1.9 8.3 4.3 13 4.3 4.7 0 9.3-2.4 13-4.3l32-24.2 52.6-39.7 106.7-79.9.3-.3c22.4-16.9 38.7-40.6 45.6-67.5 8.6-27 7.4-55.8-2.6-82z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tracking", "navigation.tabs", "navigation.tabs.sticky", "navigation.path"], "search": "../assets/javascripts/workers/search.74e28a9f.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.220ee61c.min.js"></script>
      
        
          <script src="../javascripts/mathjax.js"></script>
        
      
        
          <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
        
      
        
          <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
        
      
    
  </body>
</html>